modules/ROOT/pages/deploy-index.adoc

```diff
@@ -1,29 +1,36 @@
 = Deploy Applications and API Proxies to Runtime Fabric
 
-Anypoint Runtime Fabric enables you to deploy Mule applications and API Proxies to a Mule runtime managed on Runtime Fabric.
+Anypoint Runtime Fabric enables you to deploy Mule applications and API Proxies to a Mule runtime engine that is managed on Runtime Fabric.
 
-== Supported Mule Runtimes
+== Supported Mule Versions
 
-Anypoint Runtime Fabric supports deploying to the following versions of Mule runtime:
+Anypoint Runtime Fabric supports deployment to these versions of Mule runtime:
 
-* 3.8.7, 3.9.1, 3.9.2
-* 4.1.2, 4.1.3, 4.1.4
+* 3.8.7
+* 3.9.1, 3.9.2
+* 4.1.2+
+* 4.2.0
 
 == Prerequisites
 
-Before deploying a Mule application or API proxy to Runtime Fabric, perform the following:
+Before you deploy a Mule application or API proxy to Runtime Fabric, you must:
 
 * Install and configure a Runtime Fabric.
-* Publish your Mule application or API proxy to Exchange. Before deploying to Runtime Fabric, Mule applications and API proxies must first be published to Exchange. When deploying from Runtime Manager, this action is performed automatically.
-* Ensure that you understand how resources are allocated as described in xref:deploy-resource-allocation.adoc[Resource Allocation on Anypoint Runtime Fabric]
+* Ensure that you understand how resources are allocated as described in xref:deploy-resource-allocation.adoc[Resource Allocation on Anypoint Runtime Fabric].
+* Publish your Mule application or API proxy to Exchange. +
+[NOTE]
+Before you deploy Mule applications and API proxies to Runtime Fabric, you must first publish them to Exchange. When you deploy Mule applications and API proxies from Runtime Manager, they are published to Exchange automatically.
+
 
 == Deploy Using Runtime Manager
 
-You can manually deploy Mule applications and API proxies using Runtime Manager. See xref:deploy-to-runtime-fabric.adoc[Deploy a Mule Application to Runtime Fabric]
+You can use Runtime Manager to manually deploy Mule applications and API proxies. See xref:deploy-to-runtime-fabric.adoc[Deploy a Mule Application to Runtime Fabric].
 
 == Deploy Using Maven
 
-Runtime Fabric supports Maven for managing and deploying a Mule application or API proxy. See the following topics for information specific to the version of Mule runtime your application supports:
+Runtime Fabric supports Maven for managing and deploying a Mule application or API proxy. 
+
+The following topics have information that's specific to the version of Mule your application supports:
```

Comment:
To avoid passive voice, something like "To use Maven in this way, see the topic specific to your Mule version...."